tests/e2e_logging.py

```diff
@@ -46,6 +46,27 @@ def verify_receipt(receipt, network_cert, check_endorsement=True):
     ccf.receipt.verify(root, receipt["signature"], node_cert)
 
 
+def add_jwt(network):
```

Comment:
This does use some of the existing infra, it's just moving the functionality that was previously in `test_multi_auth` into a standalone function so it can be re-used to add JWT-auth in other tests. I'm not actually using that now, but was briefly. I'll see if we can do the equivalent (create single token, don't need a server) with `jwt_issuer.py`.

Reply:
Great catch - this entire thing can be replaced by 3 lines:

```python
jwt_issuer = infra.jwt_issuer.JwtIssuer()
jwt_issuer.register(network)
jwt = jwt_issuer.issue_jwt()
```